docs/40-components/zzz-other.mdx

```diff
@@ -31,20 +31,16 @@ Automatically bakes all non-animated blendshapes into your avatar's meshes. Redu
 
 ## Blink Controller
 
-Include a single-frame animation of your avatar with its eyes closed (or click the plus and give it the blend shape name), and VRC Fury will drive your avatar's blink cycle.
+Include a single-frame animation of your avatar with its eyes closed (or click the plus and give it the blend shape name), and VRCFury will drive your avatar's blink cycle.
 
 Benefits:
-* Blinking will stop automatically when your avatar performs vrcfury gestures affecting its eyes. This means no more 'double-blinking'.
-* Unlike vrc's built-in eye tracking disable feature, your eyes will not freeze closed, partially closed, unfreeze unexpectedly due to combo-gestures.
+* Blinking will stop automatically when your avatar performs VRCFury gestures affecting its eyes. This means no more 'double-blinking'.

```

Comment:
May as well remove this line, since it's on par with vrc's now that our gestures use tracking control.

Reply:
Which line in particular? Double-blinking?